packages/tasks/src/tasks/feature-extraction/spec/output.json

```diff
@@ -0,0 +1,7 @@
+{
+	"$id": "/inference/schemas/feature-extraction/output.json",
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	"description": "The embedding for the input text, as a nested list (tensor) of floats",
+	"type": "array",
```

Comment:
Note: it's an array in sentence transformers (one embedding per input), a list within a list in transformers (one embedding per token), and a list within a list within a list in Inference API (for batching) iirc


1. https://github.com/huggingface/api-inference-community/blob/main/docker_images/sentence_transformers/app/pipelines/feature_extraction.py#L25
2. https://github.com/huggingface/transformers/blob/main/src/transformers/pipelines/feature_extraction.py#L96